### PR TITLE
Persist with on tab model

### PR DESCRIPTION
**Pull request by lbwexler**

### CODE REVIEW

File: client-app/src/desktop/tabs/containers/TabPanelContainerPanel.js [hunk 1]
```diff
@@ -42,6 +42,7 @@ export const tabPanelContainerPanel = hoistCmp.factory({
                 height: 400,
                 item: tabContainer({
                     model: {
+                        persistWith: {localStorageKey: 'toolboxContainerTabs'},
```
Comment (lbwexler) [inline client-app/src/desktop/tabs/containers/TabPanelContainerPanel.js:45]: maybe `tabExampleState`? -- toolbox is going to be implied.  Also have been using the 'state' suffix to denote persistWith prefs and localStorage

Comment (cnrudd) [reply client-app/src/desktop/tabs/containers/TabPanelContainerPanel.js:45]: Suggestion applied.

**Comment by lbwexler** [comment]
Hi Colin -- I was able to find a good example for this in Portfolio.

We aren't really

**Comment by lbwexler** [comment]
Merged this little tweak -- nice to get some code coverage for this usage.
Hopefully we can get some better docs around persistence soon.

**Pull request merged**